.gitignore

```diff
@@ -154,3 +154,5 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+.DS_Store
```

Comment:
this goes in your `~/.gitignore` file, not the project file

Reply:
You live you learn, removed

Reply:
git config --global core.excludesfile ~/.gitignore

This was a new one to me, so thnx!